types/src/main.ts

```diff
@@ -3372,6 +3372,26 @@ export type CodeActionDisabled = {
 	reason: string;
 };
 
+/**
+ * Code action tags are extra annotations that tweak the rendering of a code action.
```

Comment:
I know we did this with other tags but we should not make the concept rendering specific. So I would say:

Code action tags are extra annotations that tweak the behavior of a code action.


For example a client could decide that LLM generated code actions must always go through a review (diff editor).